cborg/src/Codec/CBOR/Decoding.hs

```diff
@@ -315,11 +315,13 @@ getDecodeAction (Decoder k) = k (\x -> return (Done x))
 toInt8   :: Int# -> Int8
 toInt16  :: Int# -> Int16
 toInt32  :: Int# -> Int32
-toInt64  :: Int# -> Int64
 toWord8  :: Word# -> Word8
 toWord16 :: Word# -> Word16
 toWord32 :: Word# -> Word32
+#if WORD_SIZE_IN_BITS == 64
```

Comment:
For consistency, can we use the same macro as everywhere else: either `ARCH_64bit` or `ARCH_32bit`.

Reply:
Right, will also change the existing usage of `WORD_SIZE_IN_BITS` :+1: